data/account/account.go

```diff
@@ -16,8 +16,7 @@
 
 package account
 
-//go:generate dbgen -i root.sql -p account -n root -o rootInstall.go
-//go:generate dbgen -i part.sql -p account -n part -o partInstall.go
```

Comment:
the partInstall.go isn't an auto-generated file, and the part.sql doesn't exists either.